### PR TITLE
Setup 1.9.x release branch

### PR DESCRIPTION
**Pull request by bwbarrett**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by bwbarrett** [comment]
The RELEASENOTES.md is a copy of the HEAD of the v1.8.x-aws branch, with the 1.9.0 block added:

```
% git diff v1.8.x-aws v1.9.x-aws-setup -- RELEASENOTES.md                                                                                                                                                                   ✭
diff --git a/RELEASENOTES.md b/RELEASENOTES.md
index 9a566a8..cbccf40 100644
--- a/RELEASENOTES.md
+++ b/RELEASENOTES.md
@@ -11,6 +11,25 @@ AWS-specific parts a compile-time option.  When a feature (or entire
 release) was only available in one of the two variants, we note that
 in the release notes.
 
+# v1.9.0-aws release notes
+
+New Features:
+* Support v8 plugin interface introduced with NCCL 2.20.  This enables
+  the use of the user memory registration feature recently introduced
+  in NCCL.
+* Reduce ordering constraints for control messages, to reduce head of
+  line blocking under congestion.
+
+Bug Fixes:
+* Increase the number of communicators to 256K (from 1K), supporting larger
+  all-to-all groups.
+* Improve logging in some corner case error conditions.
+
+The plugin has been tested with following libfabric providers using tests
+bundled in the source code and
+[nccl-tests](https://github.com/NVIDIA/nccl-tests) suite:
+* efa
+
 # v1.8.1-aws release notes
 This is a bugfix release that requires
 [Libfabric v1.18.0](https://github.com/ofiwg/libfabric/releases/tag/v1.18.0) or later and
```

File: configure.ac [hunk 1]
```diff
@@ -6,7 +6,7 @@
 #
 
 # Initialization
-AC_INIT([aws-ofi-nccl], [GitHub-dev], [al-ofi-nccl-team@amazon.com], , [http://github.com/aws/aws-ofi-nccl])
+AC_INIT([aws-ofi-nccl], [1.9.0a1], [al-ofi-nccl-team@amazon.com], , [http://github.com/aws/aws-ofi-nccl])
```
Comment (aws-nslick) [inline configure.ac:9]: Can we make this semver compatible?

Comment (bwbarrett) [reply configure.ac]: We've historically used the more GNU-ish a1 format than the preferred semantic versioning grammar and I'm not really sure there's a good reason to change right now.  All the tools that do version number parsing are ok with the GNU-style, because GNU.

**Review by rajachan — DISMISSED** [review]

File: RELEASENOTES.md [hunk 2]
```diff
@@ -1,7 +1,601 @@
...
+
+New Features:
+* Support v8 plugin interface introduced with NCCL 2.20.  This enables
+  the use of the user memory registration feature recently introduced
+  in NCCL.
+* Reduce ordering constraints for control messages, to reduce head of
+  line blocking under congestion.
+
+Bug Fixes:
+* Increase the number of communicators to 256K (from 1K), supporting larger
```
Comment (rauteric) [inline RELEASENOTES.md:24]: Nit: (from 4K)

**Review by rauteric — DISMISSED** [review]

**Review by rauteric — APPROVED** [review]
